docs/coordinates/index.rst

```diff
@@ -278,6 +282,7 @@ listed below.
    sgr-example
    galactocentric
    definitions
+   remote_methods
```

Comment:
I would put `remote_methods` above `definitions`: the intent is for `definitions` to always be at the end as "reference material"